perseo-main/pom.xml

```diff
@@ -31,6 +31,12 @@
 
     <artifactId>perseo-main</artifactId>
 
+    <repositories>
```

Comment:
Any modification in CHANGES_NEXT_RELEASE?

Not sure if adding new entries or (if this PR is somekind or partical rollaback of PR https://github.com/telefonicaid/perseo-core/pull/143) or removing/modifying existing ones.

Reply:
Is not really a rollback, the idea is to avoid issues on previous deployments that uses those functions. 

But now I'm thinking that maybe those previous deployments shouldn't be updated having knowledge of changes on this update. What do you think about it @fgalan @AlvaroVega ?

Reply:
Fixed in 5d02f65